norse/examples/plot_mnist.py

```diff
@@ -49,30 +48,19 @@ def __init__(
             (1024,), p=LIFParameters(method=model, alpha=100.0)
         )
         self.out = LICell(1024, 10)
-        self.device = device
         self.dtype = dtype
 
     def forward(self, x):
         seq_length = x.shape[0]
         seq_batch_size = x.shape[1]
 
         # specify the initial states
```

Comment:
Since this is a tutorial I think there should be a comment here, explaining the initialisation